test/es6/moduletest2_mod0.js

```diff
@@ -0,0 +1,9 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+import './moduletest2_mod1a.js';
+import './moduletest2_mod1b.js';
+
+loaded++;
+if (loaded == 5) console.log("Pass");

```

Comment:
Empty output is a failed case. For example, when module instantiation doesn't happen, this test fails (confirmed).

---
In reply to: [107269987](https://github.com/Microsoft/ChakraCore/pull/2709#discussion_r107269987) [](ancestors = 107269987)